pkgs/development/interpreters/python/cpython/2.7/default.nix

```diff
@@ -45,6 +50,10 @@ let
     pythonVersion = with sourceVersion; "${major}.${minor}";
     sitePackages = "lib/${libPrefix}/site-packages";
     inherit hasDistutilsCxxPatch pythonForBuild;
+    pythonPackagesBuildBuild = pkgsBuildBuild.${pythonAttr};
```

Comment:
While it works for most interpreters, it won't work for variants. See e.g. `python3Minimal`. Best make `pythonAttr` an explicit string parameter. I suppose `self` would also ideally be replaced with that? That will unfortunately be breaking.